index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="top-bar">
+        <div class="form-group">
+            <img src="imgs/facebook-logo.png" class="facebook-logo">
+        </div>
+                 <div class="form-group">
+                <form class="facebook-login">
+                    <div class="form-group">
+                        <label for="email" id="user-email-phone-label">Email ou telefone</label>
+                        <input type="text" id="user-email-phone" />
+                      </div>
+                        <div class="form-group">
+                            <label for="password" id="user-password-label">Senha</label>
+                            <input type="password" id="user-password" />
+                          </div>
+                        <div class="form-control">
+                        <button id="button-login">Entrar</button>
+                        </div>
+                </form>
+            </div>
+        </div>
+    </header>
+    <main>
+        <div class="main-content">
+            <div class="left-content">
+            <p id="facebook-slogan" class="facebook-slogan-css">O Facebook ajuda você a se conectar e compartilhar com as pessoas que fazem
+                parte da sua vida.</p>
+            <img src="imgs/networking.png" alt="Facebook networking" id="facebook-networking">
+        </div>
+        <div class="right-content">
+            <h1>Abra uma conta</h1>
+            <div class="quick-easy">É rápido e fácil.
+            </div>
+            <form>
+                <div class="aolado">
+                    <input type="text" name="firstname" id="firstname" placeholder="Nome">
+                    <input type="text" name="lastname" id="lastname" placeholder="Sobrenome">
+                </div>
+                <input type="text" name="phone_email" id="phone-email" placeholder="Celular ou email">
+                <input type="password" name="password" id="password" placeholder="Nova senha">
+                <div>
+                <div>
+                    <label for="nascimento" id="label-birthdate">Data de nascimento</label>
+                <input type="text" name="birthdate" id="birthdate" placeholder="dd/mm/aaaa">
+                </div>
+                 <div>
+                <div>
+                 <label id="label-gender">Gênero </label><br>
```

Comment:
```suggestion
                 <label id="label-gender">Gênero</label><br>
```
O que acham de apagar o espaço antes do fechamento da tag `label` ?